.circleci/config.yml

```diff
@@ -693,88 +693,184 @@ jobs:
             cd js_modules/dagit
             yarn run check-prettier
 
-  airline-demo-py36: &airline-demo-template
-    <<: *dagster-template
-    docker:
-      - image: dagster/cci-test:3.6
-        auth:
-          username: $DOCKERHUB_USERNAME
-          password: $DOCKERHUB_PASSWORD
-      - image: circleci/postgres:9.6.2-alpine
-        environment:
-          POSTGRES_USER: test
-          POSTGRES_PASSWORD: test
-          POSTGRES_DB: test
+  airline-demo-py36:
+    machine:
+      docker_layer_caching: true
     environment:
       TOXENV: py36
-
+      SLUGIFY_USES_TEXT_UNIDECODE: 'yes'
+      PYTHON_VERSION: "3.6.3"
+      AIRFLOW_HOME: "/home/circleci/airflow"
+    working_directory: ~/repo
     steps:
       - checkout
-
+      - run:
+          name: Install py36
+          command: |
+            pyenv install 3.6.3
```

Comment:
it does for sure, but we need to run on circle's "machine" executor (so that we are doing what look more like bare-metal installs) to avoid the issues in https://circleci.com/docs/2.0/building-docker-images/#separation-of-environments